igcollect/linux_pressure.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+"""igcollect - PSI - Pressure Stall Information
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+import os
+import time
+
+from argparse import ArgumentParser
+
+
+def parse_args():
+    parser = ArgumentParser(prog='linux_pressure.py')
+    parser.add_argument('--prefix', default='pressure')
+    return parser.parse_args()
+
+
+def main():
+    if os.path.isdir('/proc/pressure'):
+        args = parse_args()
+        for pressure in get_pressure_stats():
+            print('{}.{}'.format(args.prefix, pressure))
+
+
+def parse_split_file(filename):
+    """Utility to read and parse a space delimited file"""
+    with open(filename, 'r') as fd:
+        return [line.strip().split(' ') for line in fd]
```

Comment:
The default of `split()` is already splitting on all whitespaces. Your `split(' ')` instead would just split on one single whitespace. So `'f  f'` would become `['f', '', 'f']`. By using the default `split()` you also don't need the `strip` beforehand. So you could change it to:
```suggestion
        return [line.split() for line in fd]
```

Reply:
Yes, you are right. Thank you, I used your shorter version now (and tested it. works!)